packages/enzyme-adapter-react-16/src/ReactSixteenAdapter.js

```diff
@@ -112,10 +113,11 @@ function toTree(vnode) {
     }
     case HostText: // 6
       return node.memoizedProps;
-    case Fragment: // 10
-    case Mode: // 11
-    case ContextProvider: // 13
-    case ContextConsumer: // 12
+    case Fragment:
+    case ContextProvider:
+    case ContextConsumer:
+    case Mode:
+    case ForwardRef:
```

Comment:
If we want to find the ForwardRef itself, we should add a node for it, not only its children here in this case.